cmd/util.go

```diff
@@ -214,25 +214,35 @@ func multiWordReplace(s string, subname string, sub string) string {
 }
 
 func getKubeVersion() string {
+	var ver string
+	var matched bool
+
 	failmsg := "kubernetes version check failed"
 	// These executables might not be on the user's path.
 	_, err := exec.LookPath("kubectl")
 	if err != nil {
-		exitWithError(fmt.Errorf("%s: %s", failmsg, err))
+		continueWithError(err, failmsg)
 	}
 
 	cmd := exec.Command("kubectl", "version", "--short")
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		exitWithError(fmt.Errorf("%s, %s", failmsg, out))
+		continueWithError(fmt.Errorf("%s", out), "")
 	}
 
-	validVersionPttn := `\d.\d`
 	serverVersionRe := regexp.MustCompile(`Server Version: v(\d+.\d+)`)
-	ver := serverVersionRe.FindStringSubmatch(string(out))[1]
+	subs := serverVersionRe.FindStringSubmatch(string(out))
+	if len(subs) > 2 {
```

Comment:
Does this work for you? I ran on a couple of deployments but it always gave me the "Unable to get kubectl version, using default version 1.6" message. I have a suspicion that this is because ver isn't getting set up because the test here should be >= 2?

Reply:
No you caught a bug